src/compas/robots/model/joint.py

```diff
@@ -609,6 +608,7 @@ def __init__(self, name, type, parent, child, origin=None, axis=None, calibratio
         self.attr = kwargs
         self.child_link = None
         self.position = 0
+        self.current_origin = Origin(self.origin.point, self.origin.xaxis, self.origin.yaxis)
```

Comment:
any reason why not use `self.origin.copy()`?

Reply:
no